shared/js/views.jsx

```diff
@@ -831,6 +927,32 @@ loop.shared.views = (function(_, mozL10n) {
     }
   });
 
+  var RemoteCursorView = React.createClass({
+    mixins: [React.addons.PureRenderMixin],
+
+    propTypes: {
+      offsetLeft: React.PropTypes.number,
```

Comment:
you mean we should set them as required? In fact, all of them are necessary to set a correct position